packages/engine-test-utils/src/__tests__/dendron-cli/commands/exportPodV2CLI.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { runEngineTestV5 } from "../../..";
+import fs from "fs-extra";
+import { ERROR_SEVERITY, Time } from "@dendronhq/common-all";
+
+
+/**
+ * Export Pod V2 
+ */
+
+const getCustomConfigPath = (wsRoot: string, podId: string) => {
+  const podsDir = path.join(wsRoot, "pods", "custom");
+  const configPath = path.join(podsDir, `config.${podId}.yml`)
+  fs.ensureDirSync(podsDir)
+  return configPath;
+}
+
+const getSvcConfigPath = (wsRoot: string, connectionId: string) => {
+  const podsDir = path.join(wsRoot, "pods", "service-connections");
+  const configPath = path.join(podsDir, `svcconfig.${connectionId}.yml`)
+  fs.ensureDirSync(podsDir)
+  return configPath;
+}
+
+// dendron exportPodV2 --podId  dendron.markdown
+describe("GIVEN export pod V2 is run ", () => {
+  describe("WHEN enrichPodArgs and --podId is used ", () => {
+      test("THEN podArgs are enriched", async () => {
+        await runEngineTestV5(
+          async ({ wsRoot }) => {
+            const podId = "dendron.markdown";
+            const configPath = getCustomConfigPath(wsRoot, podId);
+            writeYAML(configPath, {
+              exportScope: "Workspace",
+              podType: PodV2Types.MarkdownExportV2,
+              destination: "workspace-exp"
+            });
+            const cmd = new ExportPodV2CLICommand();
+            const resp = await cmd.enrichArgs({
+              wsRoot,
+              podId
+            });
+            expect(resp.data?.config.podType).toEqual(PodV2Types.MarkdownExportV2);
+            expect(resp.data?.config.exportScope).toEqual(PodExportScope.Workspace);
+            return;
+          },
+          { expect }
+        );
+    });
+  });
+
+  // dendron exportPodV2 --inlineConfig Key=podType,Value=MarkdownExport Key=exportScope,Value=Note Key=destination,Value=note-exp
+  describe("WHEN only --inlineConfig is given and exportScope is Note", () => {
+    test("THEN podArgs are enriched", async () => {
+      await runEngineTestV5(
+        async ({ wsRoot }) => {
+          const cmd = new ExportPodV2CLICommand();
+          const resp = await cmd.enrichArgs({
+            wsRoot,
+            inlineConfig: ["Key=podType,Value=MarkdownExportV2", "Key=exportScope,Value=Note", "Key=destination,Value=clipboard"],
+            fname: "root",
+            vault: "vault1"
+          });
+          expect(resp.data?.config.podType).toEqual(PodV2Types.MarkdownExportV2);
+          expect(resp.data?.config.exportScope).toEqual(PodExportScope.Note);
+          return;
+        },
+        { expect }
+      );
+    });
+  });
+
+
+
+  // dendron exportPodV2
+  describe("WHEN no config is given", () => {
+    test("THEN error must be thrown", async () => {
+      await runEngineTestV5(
+        async ({ wsRoot }) => {
+          const cmd = new ExportPodV2CLICommand();
+          const resp = await cmd.enrichArgs({
+            wsRoot
+          });
+          expect(resp.error?.severity).toEqual(ERROR_SEVERITY.FATAL);
```

Comment:
can we also check the error message? it hsould be something that is helpful for the user to make sense of